src/views/Give/DepositRow.tsx

```diff
@@ -162,14 +163,9 @@ export const DepositTableRow = ({ depositObject }: DepositRowProps) => {
           </Typography>
         </TableCell>
         <TableCell align="right" className="deposit-manage-cell">
-          <Button
-            variant="outlined"
-            color="secondary"
-            onClick={() => setIsManageModalOpen(true)}
-            style={{ width: "100%" }}
-          >
+          <SecondaryButton onClick={() => setIsManageModalOpen(true)} style={{ width: "100%" }}>
```

Comment:
not required, but you can use the fullWidth prop instead of style={{ width: "100%" }}
```suggestion
          <SecondaryButton onClick={() => setIsManageModalOpen(true)} fullWidth>
```